lang/cs/Org.Apache.REEF.Driver/Org.Apache.REEF.Driver.csproj

```diff
@@ -174,6 +174,13 @@ under the License.
   </ItemGroup>
   <Import Project="$(MSBuildToolsPath)\Microsoft.CSharp.targets" />
   <Import Project="$(SolutionDir)\.nuget\NuGet.targets" Condition="Exists('$(SolutionDir)\.nuget\NuGet.targets')" />
+  <Import Project="..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets" Condition="Exists('..\packages\StyleCop.MSBuild.4.7.49.1\build\StyleCop.MSBuild.Targets')" />
```

Comment:
This is auto-generated by adding the StyleCop.MSBuild NuGet. The package advertises that they keep the csproj in-sync with package removal or change.